config/config.go

```diff
@@ -514,6 +518,9 @@ type User struct {
 	// Name of Cache configuration to use for responses of this user
 	Cache string `yaml:"cache,omitempty"`
 
+	// Whether to allow cache for transparent_user
```

Comment:
Something happened to the indents

Reply:
actually everything is Ok, no format issues